firmware/common/portapack_shared_memory.hpp

```diff
@@ -65,6 +65,11 @@ struct SharedMemory {
         uint8_t data[512];
     } bb_data{{{{0, 0}}, 0, {0}}};
 
+    // Set by the M4 to indicate that the baseband app is ready.
```

Comment:
Using a byte as a flag to synch the M0/M4 eventing to ensure there's no race condition where the M0 tries to send events before the M4 has enabled the event that causes the event dispatcher to pump the baseband_message.